library/ix-dev/charts/nextcloud/questions.yaml

```diff
@@ -148,6 +148,14 @@ questions:
             default: 3
             min: 1
             required: true
+        - variable: opcache_memory_consumption
+          label: "Opcache Memory Consumption (Mega Bytes)"
+          description: "Configures the memory consumption of the opcache"
+          schema:
+            type: int
+            min: 128
```

Comment:
Should there be a max?

Reply:
Lets set it to 1024? I think it should be more than enough?

Reply:
I think yes